packages/neon/neon_dashboard/lib/src/app.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:neon/models.dart';
+import 'package:neon_dashboard/l10n/localizations.dart';
+import 'package:neon_dashboard/src/blocs/dashboard.dart';
+import 'package:neon_dashboard/src/options.dart';
+import 'package:neon_dashboard/src/pages/main.dart';
+import 'package:neon_dashboard/src/routes.dart';
+import 'package:nextcloud/core.dart' as core;
+import 'package:nextcloud/nextcloud.dart';
+
+/// Implementation of the server `dashboard` app.
+class DashboardApp extends AppImplementation<DashboardBloc, DashboardAppSpecificOptions> {
+  /// Creates a new Dashboard app implementation instance
```

Comment:
```suggestion
  /// Creates a new Dashboard app implementation instance.
```

Let's get the low hanging fruit :)